pkg/network/event_common.go

```diff
@@ -118,26 +118,33 @@ type BufferedData struct {
 type Connections struct {
 	BufferedData
 	DNS                         map[util.Address][]string
-	ConnTelemetry               *ConnectionsTelemetry
+	ConnTelemetry               map[ConnTelemetryType]int64
 	CompilationTelemetryByAsset map[string]RuntimeCompilationTelemetry
 	HTTP                        map[http.Key]http.RequestStats
 	DNSStats                    dns.StatsByKeyByNameByType
 }
 
-// ConnectionsTelemetry stores telemetry from the system probe related to connections collection
-type ConnectionsTelemetry struct {
-	MonotonicKprobesTriggered          int64
-	MonotonicKprobesMissed             int64
-	MonotonicConntrackRegisters        int64
-	MonotonicConntrackRegistersDropped int64
-	MonotonicDNSPacketsProcessed       int64
-	MonotonicConnsClosed               int64
-	ConnsBpfMapSize                    int64
-	MonotonicUDPSendsProcessed         int64
-	MonotonicUDPSendsMissed            int64
-	ConntrackSamplingPercent           int64
-	DNSStatsDropped                    int64
-}
+// ConnTelemetryType enumerates the connection telemetry gathered by the system-probe
+type ConnTelemetryType string
+
+//revive:disable
```

Comment:
Just curious, what was `revive` complaining about?

Reply:
Every telemetry type below needed to have a comment. I debated doing that, but I decided against it because I.think the names are pretty self explanatory 🤷‍♀️

Reply:
Ah yeah. Could you make it more specific to the rule using `revive:disable:<rulename>`?